lib/src/r_tree/node.dart

```diff
@@ -31,12 +31,7 @@ abstract class Node<E> extends RTreeContributor {
   Rectangle _minimumBoundingRect = Rectangle(0, 0, 0, 0);
 
   /// Returns the rectangle this Node covers
-  Rectangle get rect {
-    if (_minimumBoundingRect == Rectangle(0, 0, 0, 0)) {
-      updateBoundingRect();
```

Comment:
this isn't necessary anymore because we always keep bounding rect up to date? I wonder if you benchmark your branch vs master if we're recalculating bounding rect more and slowing things down when a lazy approach like master would be beneficial?

Reply:
> this isn't necessary anymore because we always keep bounding rect up to date

That is my intent.

> I wonder if you benchmark your branch vs master if we're recalculating bounding rect more and slowing things down when a lazy approach like master would be beneficial?

That's a good point. I assumed not because `rect` is used pretty extensively, but I should benchmark both of my PRs to be sure.